htlcswitch/link.go

```diff
@@ -1015,8 +1037,16 @@ func (l *channelLink) processLockedInHtlcs(
 						pd.RHash[:], l.cfg.FwrdingPolicy.MinHTLC,
 						pd.Amount)
 
-					reason := []byte{byte(lnwire.IncorrectValue)}
-					l.sendHTLCError(pd.RHash, reason)
+					var failure lnwire.FailureMessage
+					update, err := l.createChannelUpdate()
+					if err != nil {
+						failure = lnwire.NewTemporaryChannelFailure(nil)
+					} else {
+						failure = lnwire.NewAmountBelowMinimum(
+							l.cfg.FwrdingPolicy.MinHTLC, *update)
```

Comment:
The value passed here is meant to be the value of the _incoming_ HTLC, not the value of the min HTLC policy.